### PR TITLE
Fix/missing server description

### PR DESCRIPTION
**Pull request by carowright**
Missing description for servers

### CODE REVIEW

**Comment by carowright** [comment]
@TwoPlusTwoOne, now showing server description in new line under server uri

Screen Shot 2021-06-07 at 12 15 45 PM

What do you think about this changes?

**Review by twoplustwoone — DISMISSED** [review]
LGTM but since it's an important UI change @jarrodek should weigh in

File: src/ApiSummary.js [hunk 1]
```diff
@@ -397,7 +397,11 @@ export class ApiSummary extends AmfHelperMixin(LitElement) {
   _serverListTemplate(server) {
     const { baseUri, protocols } = this;
     const uri = this._computeBaseUri(server, baseUri, protocols);
-    return html`<li>${uri}</li>`;
+    const description = this._computeDescription(server);
+    return html`<li>
+      ${uri}
+      <div class="server-description">${description}</div>
```
Comment (jarrodek) [inline src/ApiSummary.js]: Can this be a markdown? If so it should also be wrapped with `arc-markdown` element.

Comment (carowright) [reply src/ApiSummary.js]: Yep, now using arc-marked for server description

**Review by twoplustwoone — APPROVED** [review]

**Pull request merged**